.github/workflows/annotate.yml

```diff
@@ -0,0 +1,58 @@
+name: Annotate checks
+
+on:
+  workflow_run:
+    workflows: ["ci", "docs"]
+    types:
+      - completed
+
+jobs:
+  annotate:
+    if: github.repository_owner == 'trinodb' && contains(['push', 'pull_request'], github.event.workflow_run.event)
```

Comment:
Won't this run on pushes to branches as well. So on PRs it'll run twice.

I guess we want something like (pseudo-code):

```suggestion
    if: github.repository_owner == 'trinodb' && (github.event.workflow_run.event == 'pull_request' || (github.even.workflow_run.event == 'push' && github.branch == 'master'))
```

Reply:
I was testing this in another repo - I removed this part, we want this to run after every `ci` workflow on both PRs and pushes.

![image](https://user-images.githubusercontent.com/795177/145585254-de1dd089-2d38-4dda-8494-5142f8239fa6.png)

The one that failed had an invalid rule.

https://github.com/nineinchnick/trino-rest/actions

Reply:
Make sense now since we already have the trigger defined as `ci, docs` `completed`.

Sorry for the noise.